### ISSUE TITLE
FFmpeg audio export fails with -22 (Invalid Argument)

### ISSUE THREAD

**Issue by generic-pers0n** [saucedacity/saucedacity#22]
**Describe the bug**
When exporting audio, `avcodec_encode_audio2` fails with a return value `AVERROR(EINVAL)` (or -22). This indicates an invalid argument being passed to `avcoded_encode_audio2()`.

**To Reproduce**
Steps to reproduce the behavior (as tested by me):
1. Record or open (obtain) some audio
2. Go to Export > Export Audio...
3. Select 'Custom FFmpeg Export
4. For the format, select 'OGG'. For the codec, select either codec available for the OGG format (either libvorbis or flac).
5. Click 'Save'
6. Fill in the metadata fields with any value (or leave them blank)
7. See error

**Expected behavior**
The export should succeed, providing completely exported audio.

**Additional information (please complete the following information):**
 - OS: Pop!_OS 21.10
 - Version: Latest Saucedacity git

**Additional context**
This issue was discovered while reorganizing FFmpeg support in Saucedacity. It was originally thought that the reorganization caused the error, but further investigation found that this error persisted in builds without any reorganization work (i.e. before the reorganization work was done).

**Comment by generic-pers0n** [comment]
Good news! FFmpeg works again in Saucedacity, after switching Saucedacity over to Audacity's `lib-ffmpeg-support`. Rather than keeping that as a library, however, it has been integrated into Saucedacity's core (i.e., `src/ffmpeg`). This issue will be closed once `lib-ffmpeg-support` is merged into `main`.

**Comment by generic-pers0n** [comment]
Oh yeah, funny coincidence I just realized: this issue (#22) involves an error code of -22. Nice.
